optaplanner-benchmark/src/main/java/org/optaplanner/benchmark/impl/result/PlannerBenchmarkResult.java

```diff
@@ -394,6 +420,8 @@ protected static PlannerBenchmarkResult createMergeSingleton(List<SingleBenchmar
                             newResult.availableProcessors, oldResult.availableProcessors);
                     newResult.maxMemory = ConfigUtils.mergeProperty(
                             newResult.maxMemory, oldResult.maxMemory);
+                    newResult.loggingLevel = ConfigUtils.mergeProperty(
+                            newResult.loggingLevel, oldResult.loggingLevel);
```

Comment:
In all other cases, loggingLevel is handled before maxMemory, so I 'll move it above that.

Great PR in general :)